README.md

```diff
@@ -1,28 +1,31 @@
 The Numenta Anomaly Benchmark [![Build Status](https://travis-ci.org/numenta/NAB.svg?branch=master)](https://travis-ci.org/numenta/NAB)
 -----------------------------
 
-Welcome. This repository contains the data and scripts comprising the Numenta
-Anomaly Benchmark (NAB). NAB is a novel benchmark for evaluating
+Welcome. This repository contains the data and scripts which comprise the
```

Comment:
I think we said we would increment the version slightly, such as V1.01 or something. I don't see that reflected in the README.